aws_batch/pipeline.py

```diff
@@ -1152,6 +1177,7 @@ def main():
     global STAR_GENOME
     global BOWTIE2_GENOME
     INPUT_BUCKET = os.environ.get('INPUT_BUCKET', INPUT_BUCKET)
+    FILE_TYPE = os.environ.get('FILE_TYPE', FILE_TYPE)
```

Comment:
I think you need to add 
global FILE_TYPE to be able to change the FILE_TYPE

Reply:
thanks

Reply:
thanks!